packages/web/src/components/organisms/EntityView/EntityView.tsx

```diff
@@ -175,8 +168,74 @@ const EntityListContent: React.FC<EntityListBlueprint> = props => {
         hasMore={!isLoadingNext && data && queryFilters.pageSize <= data.length}
         onScrollEnd={onScrollBottom}
       />
+    ),
+    [
+      CardComponent,
+      EmptyData,
+      data,
+      isFiltersEmpty,
+      isFirstTimeLoading,
+      isLoadingNext,
+      isWritable,
+      itemKey,
+      onAdd,
+      onItemAbort,
+      onItemClick,
+      onScrollBottom,
+      queryFilters.pageSize,
+      resetFilters,
+    ]
+  );
+
+  const viewComponent = useTestsSlotFirst('entityViewComponent', [{value: entityGrid, metadata: {order: 2}}]);
+
+  return (
+    <PageWrapper>
+      <PageMetadata title={pageTitle} />
+
+      <PageHeader
+        title={pageTitle}
+        pageTitleAddon={pageTitleAddon}
+        description={PageDescription}
+        loading={isApplyingFilters && !isFirstTimeLoading}
+      >
+        <PageToolbar extra={createButton}>
+          <FiltersSection>
+            <EntityViewFilters
+              setFilters={setQueryFilters}
+              filters={queryFilters}
+              disabled={isEmptyData || !isReadable}
+            />
+          </FiltersSection>
+        </PageToolbar>
+      </PageHeader>
+
+      {isValidElement(viewComponent)
+        ? cloneElement(viewComponent, {
```

Comment:
I don't like cloning elements here - it's better to use composition, so the component that will allow these props. Cloning elements is reserved for cases where actual meta-programming is needed, here it's rather case for usual composition.